sormas-app/app/src/main/java/de/symeda/sormas/app/immunization/vaccination/VaccinationEditFragment.java

```diff
@@ -50,7 +56,7 @@ public static VaccinationEditFragment newInstance(Vaccination activityRootData)
 			VaccinationEditFragment.class,
 			null,
 			activityRootData,
-			FieldVisibilityCheckers.withDisease(activityRootData.getImmunization().getDisease()),
```

Comment:
FieldVisibilityCheckers was removed because there is no field on VaccinationDto who's visibility depends on the Disease. So this check is useless here. On the other hand when creating a Vaccination having Reduced Immunization Feature active you'll have to search for the Disease throughout all the possible root entities case, contact, eventParticipant in order to get it. So I considered that this is the best approach because the above reasons.